lib/ansible/modules/cloud/ovirt/ovirt_templates.py

```diff
@@ -113,9 +120,24 @@
 # Register template
 - ovirt_templates:
   state: registered
+  storage_domain: mystorage
+  cluster: mycluster
   name: mytemplate
+
+# Register template
```

Comment:
Just please change the comment `Register template using id` or something similiar, so it's distinguished fromt the previsous example